docker/docker-compose-prod.yml

```diff
@@ -1,9 +1,9 @@
-version: "3"
+version: '3'
 
 services:
   db:
     image: postgres:9.6.16
-    container_name: burning-okr-postgres
```

Comment:
As far as I know the container_name is ignored while deploying in docker swarm, but used when deployed in a single stack. Thats why it should remain in the compose files.

Reply:
Same for the other occurrences.